### PR TITLE
DnD: startAccessor and endAccessor support

### PR DESCRIPTION
**Pull request by minznerjosh**
The DnD addon was reading event start/end times but wasn't using the startAccessor/endAccessor.

### CODE REVIEW

File: src/addons/dragAndDrop/backgroundWrapper.js [hunk 1]
```diff
@@ -107,11 +111,15 @@ function createWrapper(type) {
...
-      const { onEventDrop } = context
+      const { onEventDrop, startAccessor, endAccessor } = context
 
       onEventDrop({
         event,
-        ...getEventTimes(event, value, type)
+        ...getEventTimes({
+          ...event,
+          start: get(event, startAccessor),
+          end: get(event, endAccessor)
```
Comment (minznerjosh) [inline src/addons/dragAndDrop/backgroundWrapper.js]: `getEventTimes()` was only looking at the `start`/`end` time so spreading `...event` wasn't strictly necessary. Only did it to minimize the change.

Comment (jquense) [inline src/addons/dragAndDrop/backgroundWrapper.js]: i'd  change the signature of `getEventTimes` and avoid the spread

**Comment by jquense** [comment]
nice, thanks

**Comment by minznerjosh** [comment]
Thank YOU for this library in the first place. It's been super great working with it.

**Comment by minznerjosh** [comment]
@jquense I changed the signature.

**Pull request merged**